include/socgraph.php

```diff
@@ -1367,6 +1390,8 @@ function get_gcontact_id($contact) {
 	if ($contact["network"] == NETWORK_STATUSNET)
 		$contact["network"] = NETWORK_OSTATUS;
 
```

Comment:
please add a comment to have better documented code. the `@brief` description should do it
`// Remove unwanted parts from a contact url`